cli/command/format.go

```diff
@@ -38,7 +38,8 @@ import (
 const (
 	FORMAT_EXAMPLE = `Examples:
   $ curveadm format -f /path/to/format.yaml           # Format chunkfile pool with specified configure file
-  $ curveadm format --status -f /path/to/format.yaml  # Display formatting status`
+  $ curveadm format --status -f /path/to/format.yaml  # Display formatting status
+  $ curveadm format --stop -f /path/to/format.yaml  # stop formatting`
```

Comment:
you can use gofmt or ide auto format the code style.

Reply:
resolve ,format it
gofmt not support strings usage format ,